qt_ui/windows/basemenu/QBaseMenu2.py

```diff
@@ -124,8 +124,7 @@ def cheat_capture(self) -> None:
         self.cp.capture(self.game_model.game, for_player=not self.cp.captured)
         # Reinitialized ground planners and the like. The ATO needs to be reset because
         # missions planned against the flipped base are no longer valid.
-        self.game_model.game.reset_ato()
-        self.game_model.game.initialize_turn()
+        self.game_model.game.initialize_turn(is_reinit=True, for_player=False)
```

Comment:
This should include the player. CAS missions will be in the wrong place.

Reply:
i changed it to a complete init for both coalitions